charts/buildkitd/templates/statefulset.yaml

```diff
@@ -15,10 +15,11 @@ spec:
       labels:
         {{- include "buildkitd.selectorLabels" . | nindent 8 }}
       annotations:
-        prometheus.io/scrape: "true"
-        prometheus.io/port: {{ .Values.config.debugPort | quote }}
         container.apparmor.security.beta.kubernetes.io/buildkitd: unconfined
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        {{- if .Values.podAnnotations }}
+        {{- toYaml .Values.deployment.podAnnotations | nindent 8 }}
```

Comment:
ah sorry i forgot to change it to `{{- toYaml .Values.podAnnotations | nindent 8 }}`